### PR TITLE
Docs: Add IPv6 warning

### PR DESCRIPTION
**Pull request by ryanolee**
# What
Adds a warning about IPV6 Support in the troubleshooting guides

# Why
Sometimes routers will try to allocate either the computer / headset an IPV6 address. ALVR does not seem to be able to discover headsets when either client or server are based on a v6 address. Pending implementation of https://github.com/alvr-org/ALVR/issues/1888 or https://github.com/alvr-org/ALVR/issues/1880 it is worth adding a warning to the troubleshooting guide.

### CODE REVIEW

**Comment by Vixea** [comment]
We have already added support for multicast it's just not used by default per se.

**Comment by ryanolee** [comment]
Thanks for taking a look; Great app btw! ❤️  Might be worth leaving in until the multicast option is set as the default (Or has a toggle in the UI) from a usability / debugging standpoint? Happy to reword it to enable multicast if there is a simple way of enabling it in current versions of the app... Just currently there is no mention of IPv6 / multicast in the troubleshooting docs making it pretty awkward to debug.  

Only reason I raise it that this took me several hours of fiddling about with to pin this down myself 🥲

**Comment by Vixea** [comment]
Are you using nightly or stable?

**Comment by ryanolee** [comment]
Currently the last stable release `v20.6.1`. If this is going to be enabled / have an option in the next release can probably close this 👀

**Comment by Vixea** [comment]
Yea it's only nightly right now.

**Comment by ryanolee** [comment]
Ahh fair enough! 😆
In which probably a won't fix. Will close for now given seems like it is going to be resolved in the next few builds, thanks for the quick replies!

**Comment by Vixea** [comment]
No problem
